packages/core/src/components/reviews/EmptyReviewList/EmptyReviewList.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, Icon } from '@faststore/components'
```

Comment:
```suggestion
import { Button as UIButton, Icon as UIIcon} from '@faststore/ui'
```

**todo**: just to follow the patten

Reply:
Done in https://github.com/vtex/faststore/pull/2724/commits/13c73f19265e1275acafb18a358ea0a032de4917